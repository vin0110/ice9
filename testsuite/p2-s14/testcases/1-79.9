#1-79
fa e:= true to false -> ; af
