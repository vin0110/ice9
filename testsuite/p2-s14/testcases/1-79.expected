line 2: Semantic error: Bounds must be ints
